shiny/ui/_modal.py

```diff
@@ -158,7 +158,7 @@ def modal(
     )
 
 
-@add_example(ex_dir="../api-examples/modal")
+@add_example(ex_dir="../api-examples/modal_remove")
 def modal_show(modal: Tag, session: Optional[Session] = None) -> None:
```

Comment:
I'd prefer keeping the simpler `../api-examples/modal` example for `modal_show()`. In many cases, folks just need to show and not remove programmatically.

Reply:
sure that makes sense!